ztp/kube-compare-reference/metadata.yaml

```diff
@@ -21,6 +22,26 @@ Parts:
           - path: required/cluster-tuning/operator-hub/DefaultCatsrc.yaml
           - path: required/cluster-tuning/operator-hub/DisconnectedICSP.yaml
           - path: required/cluster-tuning/operator-hub/OperatorHub.yaml
+  - name: required-console-disable
+    Components:
+      - name: console-disable
+        type: Required
+        requiredTemplates:
+          - path: required/console-disable/ConsoleOperatorDisable.yaml
```

Comment:
Starting in 4.13 this is no longer required (disabled through capabilities). Make this optional

Reply:
Moved